data/transport missions.txt

```diff
@@ -2311,8 +2311,8 @@ mission "Terraforming 10"
 			`	Nolan answers, "The people who live near the volcano will need to evacuate the area with all their belongings. The Republic should provide any assistance needed for the evacuation. As for how we will cause the volcano to erupt, we need something energetic enough to crack the planet's crust in the location of the volcano, releasing the pressure that the magma is causing to spark an eruption. That's where we bring in another asteroi-"`
-			`	"No, no, no!" the Republic representative exclaims, cutting Nolan off mid sentence. "The Republic will not gain a reputation for crashing asteroids into its own planets. The consequences of such an action could be disastrous should it go wrong! And in what way will this warm the planet when the resulting volcanic winter could last decades and only result in an even colder climate? If the people of Tundra wish to terraform their planet that badly then normal techniques should be employed, none of this 'equilibrium mapping' nonsense."`
-			`	"Sir," Amy speaks up with a stern tone of voice, "we have already been authorized by the Republic to undergo this terraforming experiment, so unless you have half a teraton of cheap explosives, we're going to need to use an asteroid. The volcanic winter can be easily avoided by having a fleet of freighters sweep the volcanic ash out of the atmosphere, meaning we'll get the benefits of the geenhouse gases without the detriments of volcanic ash reflecting Cebalrai's heat back into space." Amy looks the representative straight in the eyes and smirks. "We're scientists, we have this handled."`
+			`	"No, no, no!" the Republic representative exclaims, cutting Nolan off mid sentence. "The Republic will not gain a reputation for crashing asteroids into its own planets. The consequences of such an action could be disastrous should it go wrong! And in what way will this warm the planet when the resulting volcanic winter could last decades and only result in an even colder climate? Are you not even old enough to remember how a supereruption on Hope caused the evacuation of the entire planet? If the people of Tundra are this desperate to terraform their planet then normal techniques should be employed, none of this 'equilibrium mapping' nonsense."`
+			`	"Sir," Amy speaks up with a stern tone of voice, "we have already been authorized by the Republic to undergo this terraforming experiment, so unless you have half a teraton of cheap explosives, we're going to need to use an asteroid. The volcanic winter can be easily avoided by having a fleet of freighters sweep the volcanic ash out of the atmosphere, meaning we'll get the benefits of the geenhouse gases without the detriments of volcanic ash reflecting Cebalrai's heat back into space. No one knew the volcano on Hope was going to erupt, but we can prepare for this eruption and take steps to midigate the problems." Amy looks the representative straight in the eyes and smirks. "We're scientists, we have this handled."`
```

Comment:
`mitigate`.

Last sentence may need a semicolon instead of a comma:
```
We're scientists; we have this handled.
```

Reply:
Or perhaps "We're scientists, we know what we're doing." or something similar?

Also, "midigate" [sic] ought to be "mitigate" and "geenhouse" [sic] "greenhouse".